packages/client-sdk-nodejs/src/internal/control-client.ts

```diff
@@ -39,6 +42,7 @@ export interface ControlClientProps {
 
 export class ControlClient {
   private readonly clientWrapper: GrpcClientWrapper<grpcControl.ScsControlClient>;
+  private readonly clientAuthWrapper: GrpcClientWrapper<grpcAuth.AuthClient>;
```

Comment:
Do we want to pull this out into its own client? The pattern that we currently have set with our clients is that they are 1:1 with a grpc service. 

https://github.com/momentohq/client-sdk-nodejs/tree/console-mvp/packages/client-sdk-nodejs/src/internal

If you look here, we have separated out `ping-service`, `data-client`, `control-client`, and `pubsub-client` into their own client files. I think we should maintain this pattern with this new auth service.

Reply:
The other thing is, do we want to have the auth client initilaize with all the other clients, or be totally separate? I am leaning towards it being totally separate because of the auth situation. We shouldn't need an auth token to initialize this client, and I don't _think_ that alot of customers will be using these endpoints directly themselves. Would be interested in what @cprice404 thinks of this

Reply:
After discussing with @cprice404 and @bruuuuuuuce, we are taking this out and create an Auth client.